src/popup/popup.js

```diff
@@ -1 +1,60 @@
-/* popup.js */
+import {
+  getAllExtensions,
+  areExtsBeingMonitored,
+  initMonitorAll,
+  stopMonitorAll,
+  viewActivityLogs,
+} from '../lib/ext-listen.js';
+
+async function init() {
+  const extensions = await getAllExtensions();
+  const startMonitorAllBtn = document.getElementById('startMonitorBtn');
+  const stopMonitorAllBtn = document.getElementById('stopMonitorBtn');
+  const viewActivityLogBtn = document.getElementById('actLogPage');
+  const monitorStatusText = document.getElementById('monitorStatus');
+
+  startMonitorAllBtn.addEventListener('click', async () => {
+    const monitorMsg = await initMonitorAll(extensions);
+    if (monitorMsg === 'ext-monitor-started') {
+      window.close();
+    } else {
+      alert("Extension monitoring couldn't be started");
+    }
+  });
+
+  stopMonitorAllBtn.addEventListener('click', async () => {
+    const monitorMsg = await stopMonitorAll();
+    if (monitorMsg === 'ext-monitor-stopped') {
+      window.close();
+    } else {
+      alert("Extension monitoring couldn't be stopped");
+    }
+  });
+
+  viewActivityLogBtn.addEventListener('click', () => {
+    viewActivityLogs();
+    window.close();
+  });
+
+  const status = await areExtsBeingMonitored();
+
+  if (status) {
```

Comment:
All of this status rendering logic should be moved to a separate function, because it is possible to toggle the status while monitoring.

Reply:
done.